Update README.md (#361)
vegaprotocol/vegawallet-browser: README.md

# Experimental browser extension

Securely connect to Vega dapps and sign transactions from your browser.

> This is an experimental release for test purposes only and is in active development. You should regard all funds transferred to keys that are managed by the extension as lost.

Reviewer: This is pretty aggressive, I think milder language would be better

```suggestion
This is an experimental release for test purposes only and is in active development. You should only transfer funds you are willing to lose to keys that are managed by the extension.
```

Installing the extension injects a Vega API for dapps and websites to interact with keys and send transactions to the Vega network. Connecting to the wallet and sending any transaction require explicit user approval.

**Understand the risks before you start**

- You retain custody of your assets
- You are responsible for the security of your assets
- You should never share your private keys
- You may encounter bugs, loss of functionality or loss of assets using the software
- No party accepts any liability for any losses whatsoever relating to its use

By using the Vega Wallet, you acknowledge that you have read and understood the [Vega Wallet User Data Policy ↗](https://vega.xyz/vega-wallet-user-data-policy/)

## Installation

> Only one extension (Fairground or Mainnet) should be installed at a time to avoid version conflict. Updates may wipe all data, and keys are not recoverable.

Reviewer: ```suggestion
Only one extension should be installed at a time to avoid version conflict. Updates may wipe all data, and keys are not recoverable. If you are communicating with multiple networks, consider using multiple browser profiles.
```
